tools/booster/Server.hs

```diff
@@ -144,17 +146,22 @@ main = do
 
             withLogger reportDirectory koreLogOptions $ \actualLogAction -> do
                 mLlvmLibrary <- maybe (pure Nothing) (fmap Just . mkAPI) mdl
-                definitions <-
+                definitionsWithCeilSummaries <-
                     liftIO $
                         loadDefinition definitionFile
-                            >>= mapM (mapM ((fst <$>) . runNoLoggingT . computeCeilsDefinition mLlvmLibrary))
+                            >>= mapM (mapM (runNoLoggingT . computeCeilsDefinition mLlvmLibrary))
                             >>= evaluate . force . either (error . show) id
-                unless (isJust $ Map.lookup mainModuleName definitions) $ do
+                unless (isJust $ Map.lookup mainModuleName definitionsWithCeilSummaries) $ do
                     flip runLoggingT monadLogger $
                         Logger.logErrorNS "proxy" $
                             "Main module " <> mainModuleName <> " not found in " <> Text.pack definitionFile
                     liftIO exitFailure
 
+                flip runLoggingT monadLogger $
+                    forM_ (Map.elems definitionsWithCeilSummaries) $ \(_, summaries) ->
+                        forM_ summaries $ \summary ->
+                            Logger.logOtherNS "booster" (Logger.LevelOther "Ceil") $ renderText (pretty summary)
+
```

Comment:
:heavy_check_mark:  This will provide the required information to the users.
However, maybe we could add the logging to the `computeCeilsDefinition` itself and run it with `runStderrLoggingT` (or with the new handle-based version), then we can use the same logging when we extend the analysis later.

Reply:
I don't see a change to the `Pretty ComputeCeilSummary` instance... so I assume the output still includes the rule?

Reply:
sorry I thought I pushed my changes yday :/

Reply:
@jberthold I double checked and I wrote a separate instance for pretty printing directly into this code as we went to have a more verbose instance when invoking with the parse binary (I think)